gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/environments/domains/UserConsentResource.java

```diff
@@ -112,7 +112,7 @@ public void revoke(
         checkAnyPermission(organizationId, environmentId, domain, Permission.DOMAIN_USER, Acl.UPDATE)
                 .andThen(domainService.findById(domain)
                         .switchIfEmpty(Maybe.error(new DomainNotFoundException(domain)))
-                        .flatMapCompletable(__ -> scopeApprovalService.revokeByConsent(domain, UserId.internal(user), consent, authenticatedUser)))
+                        .flatMapCompletable(__ -> scopeApprovalService.revokeByConsent(domain, UserId.parse(user), consent, authenticatedUser)))
```

Comment:
User.internal was good IMO as the mAPI should be aware of the GIS

Reply:
I'm still thinking this change is not required as GIS is generated for new token since more than one month and POSTMAN tests are successful in all other PR. I was more expecting a findById to get the user entity and build the UserId from this entity

Reply:
I've re-checked, looks you're correct, I've removed the `UserId.parse` and all its usages